src/import-data/ImportDataDestination.ts

```diff
@@ -362,10 +367,10 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
                   h(Fragment, [
                     isProtectedData &&
                       selectedBillingProject?.cloudPlatform === 'AZURE' &&
-                      p([
+                      div({ style: { paddingBottom: importMayTakeTime ? '1.0rem' : 0 } }, [
                         'Importing controlled access data will apply any additional access controls associated with the data to this workspace.',
                       ]),
-                    importMayTakeTime && importMayTakeTimeMessage,
+                    importMayTakeTime && div([importMayTakeTimeMessage]),
```

Comment:
"Import may take time" message, no billing project yet selected:

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/42ccb87e-9b69-4c7e-923a-47edc407e89c)

With and without the "import may take time message", no policies:

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/76725956-403a-4bad-88f1-14e54c9e1d91)


With and without the "import may take time message", with policies on the billing project:

(Content reached maximum height, scrollbar present… we may remove the height constraint)
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/e58e540c-f84a-4df7-bade-f6eda1e4610b)